utilities.py

```diff
@@ -367,3 +369,12 @@ def verify_min_length(my_dict, list_of_keys, expected_len):
         else:
             if len(my_dict[key]) < int(expected_len):
                 raise TypeError('"' + key + '" is shorter than ' + str(expected_len) + '.')
+
+def json_serial(obj):
```

Comment:
The linter is going to complain about this ... should have two blank lines between method definitions.